frontend/src/types/media.ts

```diff
@@ -94,31 +120,44 @@ export interface AudioSet {
 
 export interface AudioDetail extends Media {
   frontendMediaType: "audio"
-
+  /** An object representing a group of audio tracks this track belongs to, like an album or podcast. */
   audio_set?: AudioSet
+  /** A raw list of strings representing musical genres. */
   genres?: string[]
   length?: string
```

Comment:
I do not actually know what this is :relaxed:

Reply:
This should be removed.
`length` is a query parameter, not a property of an `AudioDetail`.

https://github.com/WordPress/openverse/blob/292946e588e839e1e305f2bf163986020b738303/frontend/src/utils/search-query-transform.ts#L38